src/core/kmod-setup.c

```diff
@@ -28,6 +28,14 @@
 #include "capability-util.h"
 #include "kmod-setup.h"
 #include "macro.h"
+#include "fd-util.h"
+#include <fts.h>
```

Comment:
please order includes alphabetically, and include global includes (i.e. those in `<>`) before local ones (i.e. those in `""`)

Reply:
(also see CODING_STYLE about that)